crates/http-api-bindings/src/embedding/mod.rs

```diff
@@ -40,6 +42,15 @@ pub async fn create(config: &HttpModelConfig) -> Arc<dyn Embedding> {
                 .expect("model_name must be set for voyage/embedding"),
             config.api_key.as_deref(),
         ),
+        "azure/embedding" => AzureEmbeddingEngine::create(
+            config
+                .api_endpoint
+                .as_deref()
+                .expect("api_endpoint is required for azure/embedding"),
+            config.model_name.as_deref().unwrap_or_default(), // Provide a default if model_name is optional
+            config.api_key.as_deref(),
+            Some("2023-05-15"),
```

Comment:
it seems that it does not need to be Optional, since now we hardcoded the api_version